src/H5detect.c

```diff
@@ -805,8 +805,6 @@ iprint(detected_t *d)
         fprintf(rawoutstream, "    * ");
         for (i = MIN(pass * 4 + 3, d->size - 1); i >= pass * 4; --i) {
             fprintf(rawoutstream, "%4d", d->perm[i]);
-            if (i > pass * 4)
-                HDfputs("     ", stdout);
```

Comment:
It's alignment for these blocks:

    /*
     *    7        6        5        4
     * SEEEEEEE EEEEMMMM MMMMMMMM MMMMMMMM
     *    3        2        1        0
     * MMMMMMMM MMMMMMMM MMMMMMMM MMMMMMMM
     * Implicit bit? yes
     */

Reply:
Ah, then the problem is that it doesn't go to `rawoutstream, but rather `stdout`.